src/backend/controllers/userController.js

```diff
@@ -479,8 +479,9 @@ exports.setWager = async (req: Request, res: Response) => {
 
 		// If a user has never made a wager or its been 24hrs since last stipend give them their stipend and update wagerDate
 		const now = new Date();
+		let newBalance = 0;
```

Comment:
If you are just using this as a boolean to check whether or not you applied the wager bonus, this is pretty confusing code. You should make it like:
`let appliedBonus=false;`
...
`appliedBonus=true;`

Reply:
Good idea